src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -483,6 +531,37 @@ private void ThrowFileNotFoundException(string errorTemplate, params object[] ar
             throw new FileNotFoundException(message);
         }
 
+        private static string s_nativeDllSubFolder;
+
+        private static string NativeSubFolderName => s_nativeDllSubFolder ?? (s_nativeDllSubFolder = GetNativeDllSubFolderName());
```

Comment:
No need to make this a property, and no need to make this a static field. Just make it an instance field and initialize it in the constructor.

Reply:
I removed NativeSubFolderName property. But NativeDllHandler() is static and I'd do not convert it to instance.